src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -15,19 +16,54 @@ import Chart from "./Chart";
 //Style
 import dashboardStyle from "./dashboardStyle";
 import { mainTheme } from "../../assets/jss/mainStyle";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import Select from "@material-ui/core/Select";
+import UploadData from "./UploadData";
 
 class Dashboard extends PureComponent {
   constructor(props) {
     super(props);
     this.state = {
       value: 0,
-    }
+      data: null,
+      error: null
+    };
   }
 
+  handleChange = (event, value) => {
+    this.setState({ value: value });
+  };
+
+  handleChangeRange = value => {
+    this.setState({ selected: value });
+  };
+
+  handleData = data => {
+    this.setState({ data });
+  };
+
+  handleError = error => {
+    this.setState({ error });
+  };
+
+  //I NEED HELP!!!!
+
   render() {
     const { classes } = this.props;
     const { value } = this.state;
     const data = (this.props.data || []).slice();
+    const selected = this.state;
+    const keys = [
```

Comment:
These keys should ideally be from the csv header, thus we shouldn't need to worry about having these values stored.